Mathlib/Data/Finsupp/NeLocus.lean

```diff
@@ -3,7 +3,7 @@ Copyright (c) 2022 Damiano Testa. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Damiano Testa
 -/
-import Mathlib.Data.Finsupp.Defs
+import Mathlib.Algebra.Group.Finsupp
```

Comment:
I don't understand the rule here; why are we moving this inside `Algebra.Group`?

Reply:
The hope is that one day we generalise `Finsupp` away from `0`. This is for example something @zeramorphic wanted for Con(NF).

With this in mind, putting a group structure on `Finsupp` belongs in the `Algebra` folder and, since it's about `AddGroup`, it should be under `Algebra.Group`.

If you have a better `Algebra.X` to suggest, I am happy to hear it.